src/chromium/crCoverage.ts

```diff
@@ -217,77 +232,27 @@ class CSSCoverage {
     ]);
     helper.removeEventListeners(this._eventListeners);
 
-    // aggregate by styleSheetId
-    const styleSheetIdToCoverage = new Map();
-    for (const entry of ruleTrackingResponse.ruleUsage) {
-      let ranges = styleSheetIdToCoverage.get(entry.styleSheetId);
-      if (!ranges) {
-        ranges = [];
-        styleSheetIdToCoverage.set(entry.styleSheetId, ranges);
+    const entries = new Map<string, CSSCoverageEntry>();
```

Comment:
These changes are scary: I'm not sure CSS coverage actually yields disjoint ranges (and IIRC we had some problems here).

Can we keep the old codepath for CSS coverage?